setup/web_server_configuration.rst

```diff
@@ -374,13 +374,6 @@ The **minimum configuration** to get your application running under Nginx is:
     After you deploy to production, make sure that you **cannot** access the ``index.php``
     script (i.e. ``http://example.com/index.php``).
 
-.. note::
-
-    By default, Symfony applications include several ``.htaccess`` files to
```

Comment:
Why not keeping it but instead document that it requires the optional package?

Reply:
Because it't already documented in the beginning of the page.